src/game/boe.main.cpp

```diff
@@ -360,6 +360,19 @@ static void replay_next_action() {
 		print_party_stats();
 	}else if(t == "display_alchemy"){
 		display_alchemy();
+	}else if(t == "display_spells"){
+		auto info = info_from_action(next_action);
+		std::istringstream sstr(info["mode"]);
+		int enum_v;
+		sstr >> enum_v;
+
+		eSkill mode = static_cast<eSkill>(enum_v);
+
+		short force_spell;
+		sstr.str(info["force_spell"]);
+		sstr >> force_spell;
+
+		display_spells(mode, force_spell, 0);
```

Comment:
Since you're changing this anyway, maybe fix the use of 0 as `nullptr` while you're at it?